test/utils.test.ts

```diff
@@ -0,0 +1,21 @@
+import { tsUnionOf } from "../src/utils";
+
+describe("utils", () => {
```

Comment:
These tests aren’t really necessary as they are all tested by proxy with other tests, and these are such simple functions with such simple inputs/outputs they have clear and obvious failures.

However, the coverage is appreciated and it doesn’t hurt to have. Thank you.